targets/TARGET_Realtek/TARGET_AMEBA/TARGET_RTL8195A/device/TOOLCHAIN_ARM_STD/sys.cpp

```diff
@@ -0,0 +1,63 @@
+/* mbed Microcontroller Library - stackheap
+ * Setup a fixed single stack/heap memory model, 
+ * between the top of the RW/ZI region and the stackpointer
+ *******************************************************************************
+ * Copyright (c) 2016 Realtek Semiconductor Corp.
```

Comment:
If possible I would follow what most of the files define (the last section for instance from SPDX website: https://spdx.org/licenses/Apache-2.0.html#licenseText)